apps/public/src/app/(map)/locatie/kies/_components/Combobox/Combobox.module.css

```
@@ -0,0 +1,26 @@
+.combobox {
+  position: relative;
+  width: 100%;
+}
+
+.comboboxOptions {
+  background-color: white;
+  position: absolute;
+  width: 100%;
+
+  /*
+      Leaflet uses z-indexes to stack layers. The highest Leaflet z-index is 700. And the SidePanel uses 702.
+      https://leafletjs.com/reference.html#map-pane
+    */
```

Imo it's enough to mention the elements with the highest z-index value here:

```suggestion
  /* The ComboxBoxOptions need to be on top of the AssetList and submit button, which have a z-index of 702. */
```